CODEOWNERS

```
@@ -1,3 +1,3 @@
 # see https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file
 
-* @HERECODEOWNERS
+* @gunzip @burnedmarshal @balanza @AleDore
```

```suggestion
* @AleDore @gunzip @burnedmarshal @balanza
```

first one is the maintainer